src/fcomputation.lisp

```diff
@@ -96,7 +97,21 @@ Create a future with:
 (defun complete-p (future)
   "Is `future` completed? Returns either `t` or `nil`."
   (with-slots (promise) future
-    (promise-finished-p promise)))
+    (or (promise-finished-p promise)
+        ;; QUESTION:
```

Comment:
That's OK. error result can be considered completed.

Reply:
Please add a test-case for the changes.

Reply:
Added complete-with-error test.